src/main/java/ca/gc/cyber/ops/assemblyline/java/client/clients/ProxyProperties.java

```diff
@@ -0,0 +1,26 @@
+package ca.gc.cyber.ops.assemblyline.java.client.clients;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.validation.annotation.Validated;
+
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
```

Comment:
Uneeded imports.  What IDE do you use?  I use intellij and they have a sonarlint plugin that I like to use, it helps pick up on small issues like this before commiting